### ISSUE TITLE
Add support for Azure Container Apps

### ISSUE THREAD

**Issue by cmaneu** [microsoft/NubesGen#322]
Waiting for Terraform support of ACA

**Comment by SandraAhlgrimm** [comment]
https://learn.microsoft.com/en-us/samples/azure-samples/container-apps-azapi-terraform/container-apps-azapi-terraform/

**Comment by cmaneu** [comment]
Hey @SandraAhlgrimm,
Great catch! Would you like to work on this task? If so, this is were I would start:
- [ ] Test the mentioned sample
- [ ] Review the existing TF modules (I don't remember your level of fluency about Terraform)
    - Having the proper
- [ ] Remove Dapr-specific things. Dapr support is interesting, but not all ACA users are DAPR users. Let's ship a simple version first 
- [ ] Look at our existing Application Insights template. I think these ones are outdated. It would make sense to create a new Azure Monitor / AI TF module, and leverage it in this ACA module.

What do you think?

**Comment by SandraAhlgrimm** [comment]
Yes, please assign it to me. I’d love to get through the „nubesgen add a
new feature“ process first before thinking about the refactoring you
suggested.

Christopher MANEU ***@***.***> schrieb am Mi. 16. Nov. 2022
um 17:09:

**Comment by jdubois** [comment]
Yes @SandraAhlgrimm this is clearly the best approach! And ACA support would help a ton of people.

**Comment by SandraAhlgrimm** [comment]
HI @cmaneu, let's wait until ACA is in the official azurerm terraform project https://github.com/hashicorp/terraform-provider-azurerm/pull/18008

It should be there soon, and it will ease adding this feature. Otherwise, we'll have to update it anyhow as soon as it gets released.
